extension.js

```diff
@@ -80,7 +80,7 @@ export default class DashToDockExtension extends Extension.Extension {
         if (!this._isEnabled || this._statusNotifierWatcher)
             return;
 
-        if (this._watchDog.nameAcquired && this._watchDog.nameOnBus)
+        if (this._watchDog?.nameAcquired && this._watchDog.nameOnBus)
```

Comment:
Mh, thinking again... I feel we should instead doing something like:

```suggestion
        if (!this._watchDog)
            return;

        if (this._watchDog.nameAcquired && this._watchDog.nameOnBus)
```

As if the watchdog is not set then we should really not pass it later to the status notifier watcher...

Reply:
That is way cleaner yup, will do that soon